### PR TITLE
Use `Enum` for setting up type and format for data export

### PR DESCRIPTION
**Pull request by tachikoma-li**
Remove global folder and use `Enum` for setting up type and format for exported data.
For backward compatible consideration, we still use the values of Enum (string) for checking function input.
We may expose the `Enum` class and then fix this issue.

### CODE REVIEW

File: qctrlopencontrols/utils.py [hunk 1]
```diff
@@ -79,3 +81,33 @@ def create_repr_from_attributes(class_instance=None, attributes=None):
     repr_string += ")"
 
     return repr_string
+
+
+class FileFormat(Enum):
+    """
+    Define exported file format.
+
+    Currently only support Q-Ctrl expanded format.
```
Comment (charmasaur) [inline qctrlopencontrols/utils.py:90]: Q-CTRL

File: qctrlopencontrols/utils.py [hunk 2]
```diff
@@ -79,3 +81,33 @@ def create_repr_from_attributes(class_instance=None, attributes=None):
...
+    CSV = "CSV"
+
+
+class Coordinates(Enum):
+    """
+    Define coordinate system for data representation.
+    """
+
+    CARTESIAN = "cartesian"
+
```
Comment (charmasaur) [inline qctrlopencontrols/utils.py:112]: Remove blank line

File: qctrlopencontrols/driven_controls/driven_control.py [hunk 3]
```diff
@@ -369,9 +368,8 @@ def _qctrl_expanded_export_content(self, file_type, coordinates):
         control_info = None
         amplitude_x = self.amplitude_x
         amplitude_y = self.amplitude_y
-        if coordinates == CARTESIAN:
-            if file_type == CSV:
-
+        if coordinates == Coordinates.CYLINDRICAL.value:
+            if file_type == FileType.CSV.value:
```
Comment (charmasaur) [inline qctrlopencontrols/driven_controls/driven_control.py:372]: I guess this was a problem before too, but could you update the docstring of this function to capitalise "CSV" and "JSON"?

File: qctrlopencontrols/driven_controls/driven_control.py [hunk 4]
```diff
@@ -497,39 +498,45 @@ def export_to_file(
         ArgumentsValueError
             Raised if some of the parameters are invalid.
         """
+        _file_types = [v.value for v in FileType]
```
Comment (charmasaur) [inline qctrlopencontrols/driven_controls/driven_control.py:501]: Could you change "Cylindrical" and "Cartesian" in the docstring to be all lower case?

File: qctrlopencontrols/utils.py (hunk 2 above)
Comment (tachikoma-li) [reply qctrlopencontrols/utils.py:112]: done

File: qctrlopencontrols/utils.py (hunk 1 above)
Comment (tachikoma-li) [reply qctrlopencontrols/utils.py:90]: done

File: qctrlopencontrols/driven_controls/driven_control.py (hunk 3 above)
Comment (tachikoma-li) [reply qctrlopencontrols/driven_controls/driven_control.py:372]: done

File: qctrlopencontrols/driven_controls/driven_control.py (hunk 4 above)
Comment (tachikoma-li) [reply qctrlopencontrols/driven_controls/driven_control.py:501]: done.

**Pull request merged**